Source/ARTTokenParams.h

```diff
@@ -14,10 +14,16 @@
 
 ART_ASSUME_NONNULL_BEGIN
 
+@protocol ARTDefaultTokenParams <NSObject>
```

Comment:
You're right, the protocol is not a good solution. You can access the `timestamp` with a simple hack `performSelector("getTimestamp")`.

Initially I was thinking to just assign nil to the `timestamp` on the `defaultTokenParams` setter but then I faced failures with [this spec](https://github.com/ably/ably-ios/blob/61ccf4ba2fc01911bba38ab115ea8931cb890d88/Spec/Auth.swift#L2163-L2176). Now I was looking to the 0.8 specs and I can't find anything related with _TokenParams .timestamp should be generated at the getter and stick_.